pynestml/codegeneration/resources_nest/directives/GSLDifferentiationFunction.jinja2

```diff
@@ -13,19 +13,19 @@ extern "C" inline int {{neuronName}}_dynamics(double, const double ode_state[],
     {%- for function in utils.get_alias_symbols(ode) -%}
       {%- if not function.is_equation() -%}
         {% set declaringExpression = function.get_declaring_expression() -%}
-  double {{names.name(function)}} = {{printerGSL.print_expression(declaringExpression)}};
+  double {{names.name(function)}} = {{printerGSL.print_expression(declaringExpression, prefix="node.")}};
```

Comment:
I think this is what makes the compilation crash (that or the next one): one needs to differentiate between predefined function and the class methods